### DISCUSSION TITLE
How to resize bottons?

### DISCUSSION THREAD

**Discussion by flobee** [osmandapp/OsmAnd#14313] [Q&A]
Hi

I'm starting using osmand for navigation on the motorbike.

one of the bigest problems: E.g. the zoom buttons, 'My location' button  and/or the direction icon (where is North which implies rotation the map: 'Map orientation & Compass' Icons)) are to small to get them touched when using gloves. double size would help. Can i change it myself in any config?

**Comment by arncor** [comment]
Hello, same usage here. I can only change their size via phone accessibility settings, which impact all apps, so I don't use it.

**Comment by scaidermern** [comment]
This has already been requested in #6997.
